crates/cheatnet/src/runtime_extensions/forge_runtime_extension/contracts_data.rs

```diff
@@ -12,13 +13,17 @@ use std::collections::HashMap;
 #[derive(Debug, Clone)]
 pub struct ContractsData {
     pub contracts: HashMap<String, StarknetContractArtifacts>,
+    pub sierra_paths: HashMap<String, Utf8PathBuf>,
```

Comment:
```suggestion
    pub source_sierra_paths: HashMap<String, Utf8PathBuf>,
```
aliases would be useful as well

Reply:
But imo it would be better to make it
```
HashMap<String, ContractData>

struct ContractData {
path
artifact
}
```

Reply:
I need class hashes to map to names during name retrieval, so I would rather leave it this way